beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -1878,6 +1880,37 @@ where
 
         (honest_head, faulty_head)
     }
+
+    pub fn process_sync_contributions(
+        &self,
+        sync_contributions: HarnessSyncContributions<E>
+    ) -> Result<(), SyncCommitteeError> {
+
+        let mut verified_contributions = Vec::with_capacity(sync_contributions.len());
+
+        for (_, contribution_and_proof) in sync_contributions {
+            if let Some(signed_contribution_and_proof) = contribution_and_proof {
+                let verified_contribution = self
+                    .chain
+                    .verify_sync_contribution_for_gossip(signed_contribution_and_proof)?;
+
+                verified_contributions.push(verified_contribution);
+
+            } else {
+                return Err(SyncCommitteeError::ContributionError(ContributionError::VerificationFail));
```

Comment:
I don't think we need this error here, it's possible that due to bad luck we end up without any sync aggregators. We could `unwrap` the `contribution_and_proof` instead, seeing as this should only be used in tests